tensorrt_llm/serve/openai_disagg_service.py

```
@@ -174,6 +316,15 @@ async def _send_disagg_request_ctx_first(
             gen_response = await self._gen_client.send_request(
                 gen_req, server=gen_server, hooks=hooks, req_id=gen_reservation_id
             )
+            if not self._warm_ctx_enabled:
+                return gen_response
+            if request.stream:
+                return self._warming_stream(
+                    gen_response, ctx_server, gen_server, gen_req, disagg_request_id
+                )
+            asyncio.create_task(
+                self._warm_ctx(ctx_server, gen_server, gen_req, disagg_request_id,
+                               self._gen_token_ids_of(gen_response)))
```

Ruff (0.16.1)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)